frontend/src/types/media.ts

```diff
@@ -95,30 +179,52 @@ export interface AudioSet {
 export interface AudioDetail extends Media {
   frontendMediaType: "audio"
 
+  /** An object representing a group of audio tracks this track belongs to, like an album or podcast. */
   audio_set?: AudioSet
+
+  /** A raw list of strings representing musical genres. */
   genres?: string[]
+
   length?: string
-  /** the time period of the track in milliseconds */
+
+  /**
+   * The time period of the track in milliseconds.
```

Comment:
```suggestion
   * The duration of the track in milliseconds.
```
Duration seems clearer than "time period" here